lunes_cms/cms/admins/document_resource.py

```diff
@@ -0,0 +1,92 @@
+from django.utils.translation import ugettext_lazy as _
+from import_export import fields, resources
+from import_export.admin import ExportActionMixin
+
+from ..models import Document
+from ..models.static import Static
+
+
+class DocumentResource(resources.ModelResource):
+    """
+    Resource to export Documents from the discipline view.
+    """
+
+    # pylint: disable=pointless-statement
+    ExportActionMixin.export_admin_action
+
+    # pylint: disable=super-init-not-called
+    def __init__(self, for_profession=None):
+        self.for_profession = for_profession
+        self.relevant_training_sets = (
+            for_profession.get_nested_training_sets() if for_profession else None
+        )
+
+    word = fields.Field(column_name=_("Word"), attribute="word")
+
+    word_type = fields.Field(column_name=_("Word type"), attribute="word_type")
+
+    article = fields.Field(
+        column_name=_("Article"),
+        attribute="article",
+    )
+
+    def dehydrate_article(self, document):
+        """
+        Method to show actual article and not their integer.
+        """
+        for x in Static.article_choices:
+            if x[0] == document.article:
```

Comment:
```suggestion
        for x in Static.singular_article_choices:
            if x[0] == document.singular_article:
```

Reply:
This is resolved now and the plural article was also added as requested by the lunes orga team :)